### ISSUE TITLE
set backgroundColor

### ISSUE THREAD

**Issue by erikaperugachi** [react-native-segmented-control/segmented-control#127]
Try to set backgroundColor={'white'}, in iOS doesn't show white color (you can see light gray).

On Android is ok!

**Comment by mike-skwi** [comment]
Any update on this? I'm still having this issue.

**Comment by DakshChhokra** [comment]
Yeah still have the same issue - Any ideas of what we can do to mitigate this @harisbaig100 ? Thanks!

**Comment by christophemenager** [comment]
I do have the same issue, with version 1.4.0

**Comment by Bamatthieu** [comment]
any update ?

**Comment by srithar21** [comment]
Hi All, I am facing this issue on React native 0.69.2 and segemented control 2.4.0. Any fix on ths?

**Comment by riledv** [comment]
I have the same problem

Looks like a **much needed thing that is missing**

@Naturalclar can you please help us?

**Comment by alexpchin** [comment]
The issue is that the SegmentControl.ios.js is. taking precedence over SegmentControl.js. You can just rename this package to use the Android version and use patch-package.

I'm not actually sure why we need both?

**Comment by VictorChen1111** [comment]
Same issue with the white color

**Comment by MattFixesFaults** [comment]
Same issue in 2025

**Comment by djopet00** [comment]
Is it possible to fix the iOS version so that styles like backgroundColor passed via props aren't overridden? It would be great to allow custom styling without having to patch the package ?

**Comment by mateofowlerz** [comment]
it's even possible to have the native segmented picker with a white background?

**Comment by Amirbeek** [comment]
It’s been 6 years. Please let us know when this will be fixed.

**Comment by UnlikePaladin** [comment]
Still an issue unfortunately and even more noticeable since Liquid Glass was added

**Comment by sixteen17777** [comment]
any changes ?
